assets/js/components/organisms/CandidateMatchCandidate.jsx

```diff
@@ -14,27 +15,41 @@ class CandidateMatchCandidate extends Component {
     }
   }
 
+  toggleCategories() {
+    this.setState({
+      showCategory: !this.state.showCategory
+    })
+  }
+
   render() {
     return (
       <Card style={{marginBottom: '1em'}}>
-        <CandidateMatchProfileBadge
-          candidateName={this.props.candidateName}
-          compositeMatchScore={this.props.compositeMatchScore} />
-
-        {
-          this.state.showCategory ?
-          this.props.categoryMatchScores.map((category, index) => {
-            return (
-              <CandidateMatchCategory
-                key={index}
-                categoryName={category.categoryName}
-                categoryMatch={category.categoryMatch}
-                questions={category.questions} />
-            )
-          })
-          :
-          null
-        }
+        <div style={{display: 'flex'}}>
+          <CandidateMatchProfileBadge
+            candidateName={this.props.candidateName}
+            compositeMatchScore={this.props.compositeMatchScore}
+            style={{flex: 1}} />
+          <Button onClick={event => this.toggleCategories()}>
+            <Glyphicon glyph="chevron-down" />
```

Comment:
You can use an inline ternary here to pick the icon displayed.  If `this.state.showCategory` is true, the icon `chevron-up`should be displayed. If false, display `chevron-down`.